cardea/cardea.py

```diff
@@ -0,0 +1,221 @@
+import json
+from inspect import isclass
+
+import featuretools as ft
+import pandas as pd
+
+import cardea
+from cardea.data_loader import EntitySetLoader
+from cardea.featurization import Featurization
+from cardea.modeling import Modeler
```

Comment:
we would need to edit [`modeling/__init__.py`](https://github.com/D3-AI/Cardea/blob/cardea-class/cardea/modeling/__init__.py) to fix this import.